learn/security/tenant_tokens.md

```diff
@@ -66,18 +66,18 @@ const tokenPayload = {
       'filter': `user_id = ${currentUserID}`
      }
   },
-  apiKeyPrefix: apiKey.substring(0, 8),
+  apiKeyUid: apiKey.substring(0, 8),
```

Comment:
@guimachiavelli: The `apiKeyUid` is now the `uid` of a valid Meilisearch API key instead of the first 8 characters. How would this line change? 😅

Reply:
I believe it should use the `uid`, no? It's probably best to check with someone from core or integrations, though.

Reply:
Hi @meilisearch/core-team, could you please lmk how this line would change? Thanks

Reply:
Since it's JS, shouldn't we call @bidoubiwa instead? Sorry for the ping Charlotte if I'm wrong
Otherwise @gmourier can help

Reply:
Thanks @curquiza ! 

This is the new usage of the generateToken method in the javascript repo: https://github.com/meilisearch/meilisearch-js/pull/1268

Reply:
Thanks @curquiza !

`apiKeyUid` does not require to be splitted on the first 8 chars. 

The `uid` of the API key is required so you should also define it in the constants above:

```js
const jwt = require('jsonwebtoken');

const apiKey = 'my_api_key';
const apiKeyUid = '7d97c2d2-bb87-48be-8e40-bae5a45696a4' // here
const currentUserID = 'a_user_id';
```

Or show the user how to fetch the key. Example here: https://github.com/meilisearch/meilisearch-js/blob/c44f4d8c875cfe9fc255e20e48adeb60e22d268f/tests/token.test.ts#L155-L158

After which in the payload there is no need for a substring: 

```js
const tokenPayload = {
  searchRules: {
    'patient_medical_records': {
      'filter': `user_id = ${currentUserID}`
     }
  },
  apiKeyUid: apiKeyUid,
  ...
}
```

Reply:
Thank you! 🙏